cubedash/summary/_model.py

```diff
@@ -191,3 +206,34 @@ def footprint_srid(self):
 def _has_shape(datasets: Tuple[Dataset, Tuple[BaseGeometry, bool]]) -> bool:
     dataset, (shape, was_valid) = datasets
     return shape is not None
+
+
+# chain all the polygon within Multipolygon into a list
+def _polygon_chain(valid_geometries):
+    polygonlist = []
```

Comment:
I like that these are stand-alone functions! :+1:  ... the core function is getting large.


Best to use docstrings, even though it's private. Lots of tools (and editor commands) touch them.
```suggestion
def _polygon_chain(valid_geometries: Iterable[BaseGeometry]) -> list:
    """Chain all the given [Mutli]Polygons into a single list."""
    polygonlist = []
```